conf/pipeline/rnaseq/utd_sysbio.config

```diff
@@ -0,0 +1,13 @@
+process {
```

Comment:
Ah also:

I would suggest also adding the whole `config_description` parameter stuff as well to the pipeline specific config, to make sure users known that there is an underlying change from the `RNASEQ` default _specifically_ for that pipeline a la

https://github.com/nf-core/configs/blob/9cca0eaf85e0ba12835c2cdd62fba12c643fec4a/conf/pipeline/eager/shh.config#L3-L8